docs/source/fitting/fitfunctions/FickDiffractionSQE.rst

```diff
@@ -0,0 +1,36 @@
+.. _func-FickDiffusionSQE:
+
+================
+FickDiffusionSQE
+================
+
+.. index:: FickDiffusionSQE
+
+Description
+-----------
+
+This fitting function models the dynamic structure factor
+for a particle undergoing jump diffusion using Fick's law for diffusion.
+
+.. math::
+
+   S(Q,E) = Height \cdot \frac{1}{\pi} \frac{\Gamma}{\Gamma^2+(E-Centre)^2}
+
+   \Gamma = DiffCoeff \cdot Q^{2}
+
+where:
+
+-  :math:`Height` - Intensity scaling, a fit parameter
+-  :math:`DiffCoeff` - diffusion coefficient, a fit parameter
+-  :math:`Centre` - Centre of peak, a fit parameter
+-  :math:`Q` - Momentum transfer, an attribute (non-fitting)
+
```

Comment:
Could you please add a reference to this fit function? Then one can do more reading if they have questions or want to know more about this function.

Reply:
This is a basic equation which appears in many book & papers. For example, Bee's book on QEENS and Bee, Chemical Physics 292 (2003) 121–141